init.rb

```diff
@@ -1,16 +1,20 @@
 require 'redmine'
 
+Rails.configuration.to_prepare do
+  Redmine::Activity.register 'evmbaseline'
+  Redmine::Search.available_search_types << 'evmbaseline'
+end
+
 Redmine::Plugin.register :redmine_issue_evm do
   name 'Redmine Issue Evm plugin'
   author 'Hajime Nakagama'
   description 'This is a plugin for Redmine. Earned value management using the ticket of redmine.'
-  version '3.5.4'
+  version '3.5.5'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.